README.md

```diff
@@ -43,19 +44,24 @@ Start the docker container and wait for the migration to conclude:
 ```bash
 docker-compose up -d
 ```
-### Step 5: Run the Application
+### Step 5: Apply Database Migrations (Alembic)
 
-Run the FastAPI application using Uvicorn:
 
-```bash
-uvicorn app:app --reload
-```
-### Step 6: Apply Database Migrations (Optional)
+1.  **Configure Alembic -** Open `alembic.ini` and set your database URL:
```

Comment:
You can set this dynamically  adding environment variables parsing logic to the 'migrations/env.py'  script. 


here  is snippet for reference

```
import os
from dotenv import load_dotenv

db = {'host': None, 'database': None, 'username': None, 'password': None}
for k in db.keys():
    val = os.environ.get(k)
    if not val:
        raise ValueError(f'enviroment variable {k} is not set')
    db[k] = val

sqlalchemy_url = f'postgresql+psycopg2://{db["username"]}:{db["password"]}@{db["host"]}/{db["database"]}'
config.set_main_option('sqlalchemy.url', sqlalchemy_url)

```